src/backend/controllers/marketController.js

```diff
@@ -67,8 +67,8 @@ exports.addMarketSale = async (req: Request, res: Response) => {
                 amount
             });
 
-            // Mark tank as being sold and save change to DB
-            tank.userId = 'Marketplace';
+            // Mark tank as being sold (give temporary ownership to master account) and save change to DB
```

Comment:
This is weird. Can't we just have the userId be from the user selling it?

Reply:
Post merge comment, the reason why we do that is because you shouldn't be able to access a tank that you're selling.